tests/test_copy_paste.py

```diff
@@ -4,10 +4,14 @@
 import random
 import os
 import platform
+import subprocess
 
 import sys
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
+project_root = os.path.join(os.path.dirname(__file__), '..')
+sys.path.insert(0, project_root)
+os.environ['PATH'] = os.path.join(project_root, 'bin') + os.pathsep + os.environ['PATH']
```

Comment:
This line isn't necessary if you run `setup.py develop`, but that might be an annoying requirement. Also I don't think it'll be enough on windows, since you'd probably need a `pyperclip.cmd` or something.